src/main/resources/campaign_finance.avro

```diff
@@ -0,0 +1,41 @@
+{
+        "namespace": "datagen",
+        "name": "campaign_finance",
+        "type": "record",
+        "fields": [
+                {"name": "time", "type": {
+                  "type": "long",
+                  "arg.properties": {
+                        "range": {
+                            "min": 1587715775521,
+                            "max": 1619273364600
+                        }
+                  }
+                }},
+                {"name": "cand_id", "type": {
```

Comment:
what does `cand_id` stand for? Should we rename it to something intuitive?

Reply:
👍 and done in latest commit.